pkg/resource/parameter_group/sdk.go

```diff
@@ -258,6 +266,13 @@ func (rm *resourceManager) sdkUpdate(
 	defer func() {
 		exit(err)
 	}()
+	if delta.DifferentAt("Spec.Tags") {
+		err = rm.updateTags(ctx, desired, latest)
+		if err != nil {
+			return nil, err
+		}
+	}
+
```

Comment:
What if a resource had only changed Tags, do you still want line 290 to be called? :)